client/src/utils/formatPrice.js

```diff
@@ -1,5 +1,9 @@
 export default function formatPrice(price) {
   const number = price.replace(/[^0-9]/g, '');
   const convertPrice = number.toString().replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ',');
```

Comment:
으음... 정확하게 어떤 상황에서 예외처리가 되는 건지는 모르겠지만 
convertPrice가 false가 되기 전에 early return 할 수 있어 보입니다. 
저도 주워들은 거지만 현업에서는 예외 처리를 위에서 처리시켜
쓸모 없는 코드 분기를 줄인다고 하더라고요!

```suggestion
export default function formatPrice(price) {
  if(!price) return;
  
  const number = price.replace(/[^0-9]/g, '');
  const convertPrice = number.toString().replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ',');
  return `₩ ${convertPrice}`;
```

Reply:
number에 숫자가 하나도 입력되지 않았을 때 빈 문자열 ('') 을 리턴해주려했습니다!
종호님 말씀대로 early return이 가능한 부분이네요!!
수정하여 좀 더 간결한 코드로 만들게요! 감사합니다!